taca/analysis/analysis_nanopore.py

```diff
@@ -206,18 +271,59 @@ def get_barcode_kit(sample_sheet):
     barcode_kit = get_barcode_kit(sample_sheet)
 
 def check_exit_status(status_file):
-    # Read pipeline exit status file and return True if 0, False if anything else
+    """Read pipeline exit status file and return True if 0, False if anything else"""
     with open(status_file, 'r') as f:
         exit_status = f.readline().strip()
     return exit_status == '0'
 
+def start_anglerfish(run_dir, af_sample_sheet, output_dir):
+    """Start Anglerfish."""
+    os.makedirs(output_dir)
+    anglerfish_command = ('anglerfish.py'
+                          + ' --samplesheet ' + af_sample_sheet
+                          + ' --out_fastq ' + output_dir
+                          + ' --threads 2'
+                          + ' --skip_demux'
+                          + ' --skip_fastqc; echo $? > .exitcode_for_anglerfish')
+    try:
+        p_handle = subprocess.Popen(anglerfish_command, stdout=subprocess.PIPE, shell=True, cwd=run_dir)
+        logger.info('Started Anglerfish for run {} using: {}'.format(run_dir, anglerfish_command))
+    except subprocess.CalledProcessError:
+        logger.warn('An error occurred while starting the Anglerfish for run {}. '
+                    'Please check the logfile for info.'.format(run_dir))
+    return
+
+def copy_results_to_lims(run_dir, anglerfish_results_dir):
```

Comment:
Ah, the function name is a bit deceiving 🙂 It just copies the result from anglerfish to a location where it can be picked up by lims, so now we only have to push a button in the lims QC step to fetch the results, instead of manually uploading the file.

Reply:
```suggestion
def copy_results_for_lims(run_dir, anglerfish_results_dir):
```